Cargo.toml

```diff
@@ -4,3 +4,10 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
+bls12_381 = "0.8.0"
+group = "0.13.0"
+ff = "0.13"
+rand = "0.8.0"
+
+[dev-dependencies]
+rand = "0.8.0"
```

Comment:
I don't think you need to declare the dev-dependency if you already have it as a dependency?